platform/server/pom.xml

```diff
@@ -212,6 +212,11 @@
             <artifactId>motech-tomcat-it</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>xml-apis</groupId>
+            <artifactId>xml-apis</artifactId>
+            <version>1.4.01</version>
```

Comment:
Version should be specified in the dependency management section of the main pom